### DISCUSSION TITLE
Hagrid

### DISCUSSION THREAD

**Discussion by abisha-coder** [community/community#34972] [Other Feature Feedback, Questions, & Ideas]
Respected Sir
hope you are doing good ... I am from Pakistan , I basically want  to run your code  in pycharm and want to see the actual working of your project... Can you please help me in doing so...? 
I have cloned the code in pycharm , complete all the  requirements , download 100 images set  but the error is  about database
![Screenshot 2022-10-03 103102](https://user-images.githubusercontent.com/114820903/193641468-3179ce20-166c-4296-8a1e-28fe4570bbe3.jpg)
Can you please give me time to solve this issue , i am still facing.... 
Best Regards 
Abisha Fatima
abishazaidi3261@gmail.com
Thank you..!

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/hukenovs/hagrid? In general you should look for support options *in that repository*, if the developers see your question here that'd be a lot of luck. :sweat_smile: 

I notice the README in that repository [mentions that training data must be downloaded separately](https://github.com/hukenovs/hagrid#downloads) because it's so big. Maybe that helps?
